### ISSUE TITLE
Using workbox without workbox-cdn

### ISSUE THREAD

**Issue by manniL** [nuxt-community/pwa-module#211]
### What problem does this feature solve?

Some applications running in an isolated VM without internet access. For them, the workbox feature can't be used as it relies on external dependencies that are unreachable.

It'd be great to have an alternative for using workbox w/o a CDN.

<div align="right"><sub><em>This feature request is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/pwa-module/issues/c153">#c153</a>)</em></sub></div>

**Comment by skyrpex** [comment]
I need this too. In my case, the project uses CSP headers heavily, restricting access everywhere. We need to allow the workbox CDN and would prefer to avoid that.

**Comment by ShaggyTech** [comment]
I can write up a more detailed explanation if anyone still needs help with this, but here is a quick mobile transcribed description of how I got this to work.

I had this working at one point in my test project but it was abandoned because it didn’t perform as well (likely due to my lack of knowledge).  I’ve switched back to using the actual CDN and long-term caching the workbox scripts as they come in to the client.

Relevant Nuxt configuration:
https://github.com/ShaggyTech/nuxt2-ssr-firebase/blob/9b69e91099fbb6e5dec83537fc6b760d2dc39564/src/nuxt.config.js#L105 

Add workbox-cdn as a dev dependency for your project.

Copy the workbox folder from node_modules/workbox-cdn into your static assets.  This folder can be on the client, server, or both.  My nuxt.config workbox options are setup to have the workbox-cdn folder located at ‘/workbox-cdn/‘ relative to root dir of the client, so make sure to adjust the settings accordingly if you put the workbox-cdn scripts elsewhere.

If the scripts are served server side only (more optimal than having the client download ALL of the workbox script files, as most are not used) then you will have to use serve-static or some other means to tell the client where the workbox scripts can be retrieved from when the service worker asks the server for them.  In my experience it was also more optimal to, at minimum, provide the client with the base workbox-sw.js file from the workbox-cdn folder to get the service worker up and running more quickly.

**Comment by clexi** [comment]
Yeah, it would be nice to have a solution for locally usage of the `workbox-sw.js` over nom :) 
But the workaround from the static folder seems to work, thanks.
